### DISCUSSION TITLE
How do I contribute?

### DISCUSSION THREAD

**Discussion by m1kecraft** [3kh0/ext-remover#790] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I forked ext-remover and added an exploit.  Should I open a pull request or something?

**Comment by AshtonDavies** [comment]
You need to have the contributor permission for pull requests in this repo.

**Reply by m1kecraft** [reply]
What should I do then?  @AshtonDavies

**Reply by AshtonDavies** [reply]
@3kh0 is the one able to give you that.

**Comment by leahanderson1** [comment, marked as answer]
> I forked ext-remover and added an exploit. Should I open a pull request or something?

yes
